care/emr/api/viewsets/report/template.py

```
@@ -0,0 +1,281 @@
+        queryset = super().get_queryset()
+        if self.action == "list" and "facility" in self.request.GET:
+            facility = get_object_or_404(
+                Facility, external_id=self.request.GET["facility"]
+            )
+
+            queryset = queryset.filter(facility=facility)
+        elif self.action == "list" and "facility" not in self.request.GET:
+            queryset = queryset.filter(facility__isnull=True)
+        return queryset
+
+    def _extract_fields_from_context(self, context_class, visited=None):  # noqa: PLR0912
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `PLR0912`)

Remove unused `noqa` directive

(RUF100)